model/src/h256_hexadecimal.rs

```diff
@@ -0,0 +1,102 @@
+use primitive_types::H256;
```

Comment:
Is this module really needed? I thought that the default `Serialize` and `Deserialize` implementation did exactly this already.

Reply:
I was just mimicing what we have done for H160Wrapper... I thought this had something to do with how it is parsed from string to and from DB...